### PR TITLE
[CHK-2623] Add missing brands to openapi

### PR DESCRIPTION
**Pull request by petretiandrea**
#### List of Changes
- Refactor brand type to String instead of enum
- Added `SupportedCardLogo` enum to keep check on missing logo mapping.

#### Motivation and Context
Use of the enum requires each time a new brand is introduced to update the microservice. Also, if the brand is not supported the service responds with a failure. Using brand as string allows to keep flexibility.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api-spec/wallet-api.yaml [hunk 1]
```diff
@@ -1348,6 +1344,16 @@ components:
           type: string
       required:
         - paymentMethodType
+    CardBrand:
+      description: Payment instrument brand
+      type: string
+      enum:
+        - MASTERCARD
```
Comment (pietro-tota) [inline api-spec/wallet-api.yaml:1351]: Maybe mastercard can be removed since NPG returns MC for mastercard cards. What do you think?

Comment (petretiandrea) [reply api-spec/wallet-api.yaml:1351]: Yes I'm agree. Also in logo mapping should it be removed? Or leave it as a possible alias? (considering future refactor of brand as a string)

**Comment by petretiandrea** [comment]
@TommasoLencioni When this PR is closed the brands in the migration file will be supported

**Review by ciuffagianluca — APPROVED** [review]
LGTM 🚀

**Review by ciuffagianluca — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**